readme.md

```diff
@@ -72,11 +72,22 @@ Like the new anki stats screen? Want more? This is the addon for you.
 
 Building this with make addon works on linux and will probably work on mac 
 
-Clone this repo recursively (It contains all anki's source files, sorry)
-```
-https://github.com/Luc-Mcgrady/Anki-Search-Stats-Extended.git --recursive
+### Cloning the Repository
+
+ To clone the repository along with its submodule (Anki codebase), you can use the following command:
+
+```sh
+git clone https://github.com/Luc-Mcgrady/Anki-Search-Stats-Extended.git --recursive --depth=1 --shallow-submodules
```

Comment:
> for --depth 1 I'd say its worth the ~8mb to have a complete commit history for the addon itself.

Agreed, This will prevent the headache of dealing with `git rebase` and friends in the future.